### ISSUE TITLE
Problem displaying arrays of 'point' data types

### ISSUE THREAD

**Issue by jw35** [dbcli/pgcli#814]
## Description

Given this table as seen by psql:

```
icp=# \d pgcli_test
  Table "public.pgcli_test"
 Column |  Type   | Modifiers
--------+---------+-----------
 points | point[] |

icp=# select * from pgcli_test ;
      points
-------------------
 {"(1,2)","(3,4)"}
(1 row)
```

pgcli complains when performing the same select:

```
jw35@/tmp:icp> select * from pgcli_test;
could not convert string to float: '(1,2)'
```

I'd note that I'm new to Postgres arrays and point types and I'm finding the syntax challenging. So it's entirely possible I'm doing something stupid, but I don't think so. In particular ```select points[1]``` works as expected in psql and pgcli so I'm fairly sure I'm correctly creating an array of points.

## Your environment

- [ ] Please provide your OS and version information.

MacOS Sierra 10.12.6

- [ ] Please provide your CLI version.

1.8.1

- [ ] What is the output of ``pip freeze`` command.

```
$ pip3 freeze
appdirs==1.4.2
arrow==0.10.0
backports.csv==1.0.5
cli-helpers==0.2.3
click==6.7
configobj==5.0.6
cycler==0.10.0
graphviz==0.8.1
humanize==0.5.1
matplotlib==2.0.2
numpy==1.13.1
packaging==16.8
pandas==0.20.3
pgcli==1.8.1
pgspecial==1.8.0
prompt-toolkit==1.0.15
psycopg2==2.7.3.1
Pygments==2.2.0
pyparsing==2.2.0
python-dateutil==2.6.1
pytz==2017.2
setproctitle==1.1.10
six==1.10.0
sqlparse==0.2.3
terminaltables==3.1.0
wcwidth==0.1.7
xmltodict==0.11.0
```

p.s. Many thanks for pgcli - it's awesome and I'm really only reporting this in the hope it can be fixed because having to work on databases containing arrays of points with psql is going to be tedious...

**Comment by jw35** [comment]
Further investigation shows that plain psycopg2 does the same thing, so I think the problem may be there rather than in pgcli. I'll open the issue there and see what happens.

**Comment by jw35** [comment]
https://github.com/psycopg/psycopg2/issues/613

**Comment by fpietka** [comment]
@jw35 good news, it got fixed with latest release (1.9.0):

```
03/03/18 14:03:27 postgres@localhost:tests> create table psycopg2_test (points point[]);
CREATE TABLE
Time: 0.030s
03/03/18 14:03:28 postgres@localhost:tests> insert into psycopg2_test values ('{"(1,2)","(3,4)"}');
INSERT 0 1
Time: 0.011s
03/03/18 14:03:41 postgres@localhost:tests> select * from psycopg2_test;
+-------------------+
| points            |
|-------------------|
| {"(1,2)","(3,4)"} |
+-------------------+
SELECT 1
Time: 0.004s
```
